src/api/api.ts

```diff
@@ -0,0 +1,7 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Tip: NodeJS now (actually was in september 2023) in the version v20.6.0 (wow, it **is** the version that our project uses 🤞😎) has native support for `.env` files, so we can use it to store this `BASE_URL` and many other values, therefore we can remove those variables throughout the codebase.

ref: https://nodejs.org/pt/blog/release/v20.6.0#built-in-env-file-support